justfile

```diff
@@ -11,33 +11,34 @@ install-tools:
 	cargo install cargo-binstall
 	cargo binstall cargo-insta taplo-cli
 	cargo binstall --git "https://github.com/astral-sh/uv" uv
-
+	bun install
```

Comment:
funktioniert das lokal, wenn in der `bun.lock` die ganzen `<placeholder>` values stehen?

Reply:
ja, sind ja optional dependencies